.gitignore

```diff
@@ -273,3 +273,7 @@ tags
 .ionide
 
 # End of https://www.toptal.com/developers/gitignore/api/vim,c,c++,python,visualstudiocode,platformio
+
+.env
```

Comment:
Please put any custom additions to this file at the top before the autogenerated section (with the other stuff I added) rather than at the bottom